islandora.module

```diff
@@ -397,7 +398,14 @@ function islandora_menu() {
     'type' => MENU_NORMAL_ITEM,
     'access arguments' => array(ISLANDORA_MANAGE_DELETED_OBJECTS),
   );
-
+  $items['admin/reports/orphaned_objects/list'] = array(
+    'title' => 'Orphaned Islandora objects',
+    'description' => 'List of orphaned Islandora objects.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('islandora_manage_orphaned_objects_form'),
+    'access arguments' => array('ISLANDORA_MANAGE_ORPHANED_OBJECTS'),
```

Comment:
I don't think the quotes here are intended.

Reply:
`'ISLANDORA_MANAGE_ORPHANED_OBJECTS'` should probably be `ISLANDORA_MANAGE_ORPHANED_OBJECTS`

Reply:
Thanks! Done.